Makefile

```diff
@@ -1,15 +1,13 @@
 
-AWS_ACCOUNT_ID ?= $(shell aws sts get-caller-identity --query Account --output text)
-AWS_REGION ?= us-west-2
 RELEASE_VERSION ?= dev-$(shell git rev-parse --short HEAD)
 OCI_REPO ?= ghcr.io/kro-run/kro
 
-CONTROLLER_IMAGE ?= ${OCI_REPO}/controller:${RELEASE_VERSION}
 HELM_IMAGE ?= ${OCI_REPO}
 KO_DOCKER_REPO ?= ${OCI_REPO}/kro
 
 KOCACHE ?= ~/.ko
 KO_PUSH ?= true
+export KIND_CLUSTER_NAME ?= kro
```

Comment:
is the `export` needed here?

Reply:
Yes needed for KO, https://ko.build/configuration/